src/CreateBookPage.js

```diff
@@ -0,0 +1,102 @@
+import React, {useState, useEffect} from 'react';
+import './App.css';
+import {sendJson, loadJson} from './sendJson';
+import {BrowserRouter as Router, Switch, Route, Link} from 'react-router-dom';
+import {Book} from './Book.js';
+
+const loadAllBooks = async (onLoad, setError) => {
+  const result = await loadJson('getAllBooks');
+  if (result.error) {
+    setError(result.error);
+  } else {
+    onLoad(result);
+  }
+};
+
+export const CreateBookPage = ({currentUser}) => {
+  const [titleState, setTitleState] = useState('');
+  const [booksState, setBooksState] = useState([]);
+  const [error, setError] = useState('');
+
+  //function to submit request to create new user.
```

Comment:
create new user?

Reply:
lol its a WIP! :P

Reply:
Besides maybe I want a secret button that makes a user! Ohh man we should have that somewhere in the code. Some super secret pixel that does something awesome if you find it

Reply:
Is it still a WIP?